05-html-css/css-framework.md

```diff
@@ -18,185 +16,144 @@ The usual components are:
 
-### After Foundation:
-![after-foundation-typography](imgs/after-foundation.png)
+### After Bootstrap:
+![Bootstrap Typography](imgs/bootstrap-typography.png)
 
-As you can see above Foundation provides an set of default fonts and styles for the basic elements.  It also provides styling for lists, abbreviations, quotations and accessibility.  You can read about Foundation's typography [here](https://foundation.zurb.com/sites/docs/typography-base.html).
 
-## Built-in Foundation Classes
+As you can see above Bootstrap provides an set of default fonts and styles for the basic elements.  It also provides styling for lists, abbreviations, quotations and accessibility.  You can read about Bootstrap's typography [here](http://getbootstrap.com/docs/4.1/content/typography/).
 
-Foundation also provides a number pre-created stylings for class names you can add to common HTML elements and take advantage of ready-made stylings.  
+## Built-in Bootstrap Classes
+
+Bootstrap also provides a number pre-created stylings for class names you can add to common HTML elements and take advantage of ready-made stylings.
 
 For example, Foundation also provides a variety of ways to create buttons on your site either using anchor or button elements.  You can experiment with foundation buttons [here](https://codepen.io/adadev/pen/WzYvyo?editors=1000).
 
 ```html
-<a class="button tiny" href="#">So Tiny</a>
-<a class="button small" href="#">So Small</a>
-<button class="button" href="#">So Basic</button>
-<a class="button alert" href="#">So Basic with alert</a>
-<a class="button success" href="#">So Basic with success</a>
-<button class="button secondary" href="#">So Basic with secondary</button>
-<a class="button warning" href="#">So Basic with warning</a>
-<a class="button large" href="#">So Large and success</a>
-<a class="button expanded" href="#">Such Expand</a>
-<a class="button small expanded" href="#">Wow, Small Expand</a>
+<section>
+  <a href="#" class="btn btn-primary">Prmary</a>
+  <a href="#" class="btn btn-secondary">Secondary</a>
+  <a href="" class="btn btn-success">Success</a>
+  <a href="#" class="btn btn-danger">Danger</a>
+  <a href="#" class="btn btn-warning">Warning</a>
+  <a href="#" class="btn btn-info">Info</a>
+  <a href="#" class="btn btn-light">Light</a>
+  <a href="#" class="btn btn-dark">Dark</a>
+  <a href="#" class="btn btn-link">Link</a>
+</section>
+<section>
+  <a href="#" class="btn btn-primary btn-lg btn-block">Block level button</a>
+  <a href="#" class="btn btn-secondary btn-lg btn-block">Block level button</a>
+  <a href="#" class="btn-lg btn-primary">Primary-large</a>
+  <a href="#" class="btn-sm btn-primary">Primary-small</a>
+</section>
 ```
 
-**Experiment:** Try creating a large warning `button` element in Codepen.  Then try adding CSS to make `secondary` buttons dark green.
+**Experiment:** Try creating a large danger `button` element in Codepen.  Then try adding CSS to make `secondary` buttons dark green.
```

Comment:
The workflow for this isn't super clear, since the link above is just one word. Is there a way to make it more obvious how to get to the codepen?

I could see a student just going to codepen by themselves, but it takes a bit of work to get codepen set up to use bootstrap.